src/pages/iou/ReceiptSelector/NavigationAwareCamera.js

```diff
@@ -0,0 +1,51 @@
+import React, {useEffect, useState} from 'react';
+import {Camera} from 'react-native-vision-camera';
+import {useTabAnimation} from '@react-navigation/material-top-tabs';
+import {useNavigation} from '@react-navigation/native';
+import refPropTypes from '../../../components/refPropTypes';
+
+const propTypes = {
+    /* Forwarded ref */
+    forwardedRef: refPropTypes.isRequired,
+};
+
+// Wraps a camera that will only be active when the tab is focused or as asoon as it start to become focused.
+function NavigationAwareCamera(props) {
+    // Get navigation to get initial isFocused value (only needed once during init!)
+    const navigation = useNavigation();
+    const [isCameraActive, setIsCameraActive] = useState(navigation.isFocused);
+    // Get the animation value from the tab navigator. Its a value between 0 and the
```

Comment:
Just adding an extra line for style
```suggestion

    // Get the animation value from the tab navigator. Its a value between 0 and the
```